Dockerfile

```diff
@@ -15,5 +15,10 @@ RUN curl -Lfs "https://github.com/openshift/rosa/releases/download/$(curl -Lfs h
 RUN curl -LSs -o /usr/local/bin/opm "https://github.com/operator-framework/operator-registry/releases/download/$(curl -Lfs https://api.github.com/repos/operator-framework/operator-registry/releases/latest \
     | jq -r .tag_name)/linux-amd64-opm" \
     && chmod 0755 /usr/local/bin/opm
+RUN curl -Ls https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz | \
```

Comment:
Please take a look how the `rosa` tool is installed by passing the tar file on stdin of `tar` and extracting directly to the target bin directory. That way is simpler as you do not need to clean up with `rm` afterwards.